### PR TITLE
Add default keybinding an alias for gui/design

### PR DESCRIPTION
**Pull request by johncosker**
Should not be merged until after https://github.com/DFHack/scripts/pull/643

Creates an `alias` for `gui/dig` -> `gui/design` to preserve existing custom user hotkeys.

### CODE REVIEW

**Review by myk002 — COMMENTED** [review]
One more item of business the could be added to this PR: `gui/dig` needs a tombstone entry in docs/about/Removed.rst so the old entries in the changelog have something to resolve to. Otherwise we'll get build errors on the docs.

Ironically, the tombstone entry will cause *this* PR to fail until the scripts PR is merged due to duplicate doc labels, but don't worry about that.

**Review by myk002 — APPROVED** [review]

File: docs/about/Removed.rst [hunk 1]
```diff
@@ -192,3 +192,9 @@ Replaced with a GUI version: `gui/unit-syndromes`.
 warn-stuck-trees
 ================
 The corresponding DF :bug:`9252` was fixed in DF 0.44.01.
+
+.. gui/dig:
```
Comment (myk002) [inline docs/about/Removed.rst:196]: Ha, the fact that the PR didn't fail tipped me off that there was an issue here. Missing an underscore

Comment (myk002) [reply docs/about/Removed.rst:196]: Also please keep in alphabetical order

**Comment by myk002** [comment]
Ah, I didn't see this before, could you fix this in the scripts repo?
scripts/docs/gui/design.rst:3: WARNING: Title underline too short.

**Comment by johncosker** [comment]
https://github.com/DFHack/scripts/pull/644

**Review by myk002 — APPROVED** [review]

**Pull request merged**